tests/test_num_regression.py

```diff
@@ -0,0 +1,157 @@
+import numpy as np
+import pytest
+
+
+@pytest.fixture
+def no_regen(num_regression, request):
+    if num_regression._force_regen or request.config.getoption("force_regen"):
+        pytest.fail("--force-regen should not be used on this test.")
+
+
+@pytest.mark.xfail(reason="#3 needs investigation", strict=True)
```

Comment:
@marcelotrevisani here's the test mentioned in #3.

Reply:
Ok, I'm going to test if it will solve

Reply:
huumm, Idk my commit didn't appear here, however I just put that thing in the end

```
    import pandas as pd
    if isinstance(obtained_error_msg, pd.DataFrame):
        with pd.option_context('display.max_rows', None,
                               'display.max_columns', None):
            assert expected in obtained_error_msg
    else:
        assert expected in obtained_error_msg
```

Reply:
It will avoid to trunk the ``DataFrame`` and will show the entire information

Reply:
Weird thing is:

* We set the display options to reasonable values
* The code *works* exactly as it is in our conda envs

Reply:
Btw, "trunk" is a good one for "truncar" hehehe

Reply:
hahahahahahah, my bad

Reply:
I mean "truncated", haha
Please try this, it also worked for me
```
pd.set_option('display.max_colwidth', -1)
```